src/pages/task/metadata/Stepback.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "@emotion/styled";
+import Badge from "@leafygreen-ui/badge";
+import { InfoSprinkle } from "@leafygreen-ui/info-sprinkle";
+import { Skeleton } from "@leafygreen-ui/skeleton-loader";
+import { BaseFontSize } from "@leafygreen-ui/tokens";
+import { MetadataItem } from "components/MetadataCard";
+import { size } from "constants/tokens";
+import { TaskQuery } from "gql/generated/types";
+import { useBreakingTask } from "hooks/useBreakingTask";
+import { useLastPassingTask } from "hooks/useLastPassingTask";
+
+type Props = {
+  taskId: string;
+};
+
+/**
+ * Returns whether the task is in stepback.
+ * @param task The task to check if it is in stepback.
+ * @returns Whether the task is in stepback.
+ */
+export function isInStepback(task: TaskQuery["task"]) {
+  // The 'lastFailingStepbackTaskId' is set for all stepback tasks except the first one.
+  const hasLastStepback =
+    task?.stepbackInfo?.lastFailingStepbackTaskId?.length > 0;
+
+  // The 'nextStepbackTaskId' is only set when the next task in stepback is running/finished.
+  // This happens in the beginning of stepback or them middle of stepback. This condition is
+  // covering for the beginning of stepback.
+  const isBeginningStepback =
+    task?.stepbackInfo?.nextStepbackTaskId?.length > 0;
+
+  // If the task is in stepback or beginning stepback, it is counted as in stepback.
+  return hasLastStepback || isBeginningStepback;
+}
+
+export const Stepback: React.FC<Props> = ({ taskId }) => {
+  const { loading, task: breakingTask } = useBreakingTask(taskId);
+  const { task: lastPassingTask } = useLastPassingTask(taskId);
+
+  // If the last passing task is undefined, it means the task is the breaking task.
+  const isBreakingTask = lastPassingTask === undefined;
+
+  // The stepback is finished if there is a breaking task or we are on the last stepback task.
+  const finished = breakingTask !== undefined || isBreakingTask;
+
+  return (
+    <MetadataItem as="div">
+      <StepbackWrapper>
+        Stepback:
+        <InfoSprinkle baseFontSize={BaseFontSize.Body1}>
+          When Stepback is completed you can access the breaking commit via the
+          relevant commits dropdown.
+        </InfoSprinkle>
+        {loading && <Skeleton size="small" />}
```

Comment:
nit. loading state looks like it might need a small style tweak

<img width="295" alt="Screenshot 2024-03-21 at 12 52 29 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/716bc439-8573-408f-9a04-00b917321383">

Reply:
<img width="300" alt="Screenshot 2024-03-21 at 2 01 00 PM" src="https://github.com/evergreen-ci/spruce/assets/64446617/37b0d65c-e60c-426b-a172-c057eb9a7552">

Added no wrap and it looks like this now!